src/Resources/Resources.Test/Resources.Test.csproj

```diff
@@ -14,7 +14,6 @@
     <PackageReference Include="Microsoft.Azure.Graph.RBAC" Version="3.5.0-preview" />
     <PackageReference Include="Microsoft.Azure.Management.Authorization" Version="2.11.0-preview" />
     <PackageReference Include="Microsoft.Azure.Management.ManagementGroups" Version="1.1.1-preview" />
-    <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="2.1.0-preview" />
```

Comment:
@cormacpayne do I need to update the ResourceManager package version in TestFx and ScenarioTest.ResourceManager project?